src/base/solvers/verifyCobraProblem.m

```diff
@@ -146,7 +154,7 @@
 
 %check ub
 if ~isfield(XPproblem,'ub')
-    diplay('Required field ub not found');
+    display('Required field ub not found');

```

Comment:
Why are you using sometimes `display` and sometimes `disp`?

Reply:
because personally, I write "disp", but the erronous diplay was in the code and I just corrected it.

Reply:
Also, the code originally had a mix of it as well.